cpp_ext/TorchOps.cpp

```diff
@@ -127,21 +103,42 @@ void populateTorchMLIROps(py::module &m) {
 #include "TorchOps.pybinds.cpp"
 
   // prim::abs.Scalar : (Scalar) -> (Scalar)
-  m.def("abs", py::overload_cast<const PyAnyTorchScalarValue &>(&abs));
+  m.def(
+      "abs",
+      [](const PyAnyTorchScalarValue &x, DefaultingPyLocation &loc,
+         const py::object &ip) {
```

Comment:
The `Defaulting*` types [work by casting from `py::none()`](https://github.com/llvm/llvm-project/blob/main/mlir/lib/Bindings/Python/PybindUtils.h#L60-L68). There's `DefaultingInsertionPoint` (maybe a todo?) so `getInsertionPoint` basically does the same thing.